doxygen/dox/ReferenceManual.dox

```diff
@@ -1,5 +1,8 @@
 /** \page RM Reference Manual
 
+\todo Add the Event Sets module
+\todo Add the Map module
+
 The functions provides by the HDF5 C-API are grouped into the following
```

Comment:
I know this line isn't part of the PR, but I happened to notice a typo for "provided" here.

Reply:
Nice catch :-)